modules/apps/transfer/types/query.pb.go

```diff
@@ -34,7 +34,8 @@ const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
 // method
 type QueryDenomTraceRequest struct {
 	// hash (in hex format) of the denomination trace information.
-	Hash string `protobuf:"bytes,1,opt,name=hash,proto3" json:"hash,omitempty"`
+	Hash   string `protobuf:"bytes,1,opt,name=hash,proto3" json:"hash,omitempty"`
+	Height uint64 `protobuf:"varint,2,opt,name=height,proto3" json:"height,omitempty"`
```

Comment:
Would `QueryHeight` be a better name? Should we use uint64 or int64? Ideally all the core IBC queries are standardized across all implementations

Reply:
Will update to `QueryHeight` and keep `uint64`. QueryHeight won't cause name collisions as `Height` might